doc/Administration-guide/System-Configuration/General-configuration/index.md

```diff
@@ -19,168 +28,190 @@ $ sudo service kaa-node restart
-
-Maximum redirect probability for existing sessions.
-
-#### recalculation_period
-Default: _10_
-
-Recalculate period in seconds for Operations service [load balancer]({{root_url}}Administration-guide/System-components-overview/#load-balancing-lb) process.
-
-#### user_hash_partitions
-Default: _10_
-
-Specify consistent-hash partitions count for each server node.
-
-#### max_number_neighbor_connections
-Default: _3_
-
-Specify the max number of neighbor connections.
-
-#### ops_server_history_ttl
-Default: _3600_
-
-Time to live in seconds for historical information about Operations service load.
-
-#### worker_thread_pool
-Default: _8_
-
-Message Handler thread pool executor size.
-
-#### bootstrap_keys_private_key_location
-Default: _keys/bootstrap/private.key_
-
-Path to Bootstrap service private key.
-
-#### bootstrap_keys_public_key_location
-Default: _keys/bootstrap/public.key_
-
-Path to Bootstrap service public key.
-
-#### operations_keys_private_key_location
-Default: _keys/operations/private.key_
-
-Path to Operations service private key.
-
-#### operations_keys_public_key_location
-Default: _keys/operations/public.key_
+## Public host/port configuration
 
-Path to Operations service public key.
+Kaa server is going to be deployed in cloud service such as AWS, Google Cloud etc.
+After deployment it will have private and public IP address.
+Public IP will be available from the internet on particular port.
+Private IP will be available within virtual network in cloud service.
 
-#### support_unencrypted_connection
-Default: _true_
+Kaa allows you to specify what IP and host/port is used for public and private interface/port for communication with Bootstrap and Operations servers.
+This is useful because once public and private IPs retrieved you have ability to report public IP/port to the client in form of embedded configuration in generated SDK and specify private IP/port where netty binds and serves requests.
 
-Specify if support unencrypted connection from client to Kaa server.
+There are few configuration files for this purpose:
 
-#### transport_bind_interface
-Default: _0.0.0.0_
+* kaa-node.properties
+* operations-http-transport.config
+* operations-tcp-transport.config
+* bootstrap-http-transport.config
+* bootstrap-tcp-transport.config
 
-Interface that will be used by all transports.
+Below is default state of file kaa-node.properties:
 
-#### transport_public_interface
-Default: _localhost_
+```bash
+# Interface that will be used by all transports
+transport_bind_interface=0.0.0.0
 
-Interface that will be reported by all transports.
+# Interface that will be reported by all transports
+transport_public_interface=localhost
+```
 
-#### metrics_enabled
-Default: _true_
+Where *transport_bind_interface* reflects private IP for Bootstrap and Operations servers, and *transport_public_interface* - public accordingly.
+Below are given default config files, which consume above host values from file kaa-node.properties.
```

Comment:
please add a note that if public_interface is set to localhost - SDK will send request to localhost, so only applications that are launched on the same host will work.